tests/DDG.spec.js

```diff
@@ -0,0 +1,15 @@
+const { test, expect } = require('@playwright/test');
+
+/*test('basic test', async ({ page }) => {
+  await page.goto('https://playwright.dev/');
+  //await page.pause();
+  const title = page.locator('.navbar__inner .navbar__title');
+  await expect(title).toHaveText('Playwright');
+});*/
+
+test.only('check logo', async ({ page }) => {
```

Comment:
We have only single test, I would suggesr not using test.only here. Also, test name should be meaningful and reflect what it is doing, e.g. 'Checks that duckduckGo page can be opened'